### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by maartenvanderhoef**
for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Microservice Boilerplate
+
+[![Build Status](https://travis-ci.com/Flaconi/terraform-aws-waf-acl-rules.svg?branch=master)](https://travis-ci.com/Flaconi/terraform-aws-waf-acl-rules)
+[![Tag](https://img.shields.io/github/tag/Flaconi/terraform-aws-waf-acl-rules.svg)](https://github.com/Flaconi/terraform-aws-waf-acl-rules/releases)
+<!-- [![Terraform](https://img.shields.io/badge/Terraform--registry-aws--iam--roles-brightgreen.svg)](https://registry.terraform.io/modules/Flaconi/waf-acl-rules/aws/) -->
```
Comment (danvaida) [inline README.md:5]: this should be updated.

**Review by cytopia — CHANGES_REQUESTED** [review]
Please use proper commit messages

**Review by danvaida — DISMISSED** [review]
approving nonetheless ;) nicely done.

**Review by cytopia — APPROVED** [review]
Eyes closed, but please ensure to write more "descriptive" messages in the future.

**Pull request merged**